lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -146,6 +147,70 @@ class ConvertAten_IndexPutImplOp
   }
 };
 
+class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
+public:
+  using OpConversionPattern<AtenConvolutionOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenConvolutionOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    SmallVector<Type> resultTypes;
+    if (failed(OpConversionPattern<AtenConvolutionOp>::getTypeConverter()
+                   ->convertTypes(op->getResultTypes(), resultTypes))) {
+      return failure();
+    }
+
+    SmallVector<Value> operands;
+    operands.push_back(adaptor.getInput());
+    operands.push_back(adaptor.getWeight());
```

Comment:
Why is the `$bias` [operand](https://sourcegraph.com/github.com/llvm/torch-mlir@4dd17f0b711523223e41606d9b7b6023b9149d46/-/blob/include/torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td?L5578) (optional torch tensor) being dropped?